frontend/src/newConsole/components/newLogos/LogosSidebar.tsx

```diff
@@ -18,13 +16,9 @@ interface ActionProps {
   updateImages: Function;
 }
 interface StateProps extends InstancesModel {}
-interface Props extends StateProps, ActionProps {
```

Comment:
The back functionality is required on this page.

Reply:
Added in updating the components